platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/TcpFactory.java

```diff
@@ -47,4 +49,9 @@ public TcpFactory(@NonNull final SocketConfig socketConfig) {
         SocketFactory.configureAndConnect(clientSocket, socketConfig, hostname, port);
         return clientSocket;
     }
+
+    @Override
+    public void handlePeerListUpdate(List<PeerInfo> peerInfoList) {
+        // not implemented for TCP Factory
```

Comment:
should this throw, rather than silently doing nothing?

Reply:
It's an optional implementation. Naked tcp connections don't have a need to do this yet, and probably never will (because this entire class goes away in my next PR).